### DISCUSSION TITLE
Filter out traffic from ipv6 tunnel interface

### DISCUSSION THREAD

**Discussion by benbgg** [akvorado/akvorado#1061] [Q&A]
Hi,
1st, thanks for akvorado!
2nd. can you think of a way that I can avoid "double counting" if my ipv6 traffic arrives over a sit tunnel from Hurricane Electric?
(I'm receiving IPFIX from my Mikrotik, btw - happy to share config stanzas if anyone is interested)
My interface classifier looks like this currently:
     - |
       ClassifyConnectivityRegex(Interface.Name, "^(?i)(pppoe|sit)", "$1") &&
       ClassifyExternal()
     - ClassifyInternal()

**Comment by benbgg** [comment]
I can partially solve this problem on the "visualize" page by excluding my HE tunnel endpoint address thus: InIfBoundary = external AND NOT (Proto = 41) 
Is there a way to do this for the main page though?

**Reply by vincentbernat** [reply]
Check https://demo.akvorado.net/docs/configuration#console-service (`homepage-graph-filter`)

**Comment by benbgg** [comment]
Humm. thanks for the response, but I can't get it to work - finally tried pasting the example from docs for console section between http:... and database  into console.yaml  and otherwise leaving untouched  and I get a restart loop on orchestrator and dependencies 
docker-compose logs akvorado-orchestrator gives:
akvorado-orchestrator_1     | Error: invalid configuration:
akvorado-orchestrator_1     | invalid key "Console[0].console"****

**Comment by benbgg** [comment]
Got it! hadn't spotted console.yaml was an include in Akvorado.yaml
Thanks again
